cmd/create/cluster/cmd.go

```diff
@@ -1869,6 +1879,13 @@ func run(cmd *cobra.Command, _ []string) {
 	if args.fakeCluster {
 		props = append(props, properties.FakeCluster)
 	}
+	if args.useLocalCredentials {
+		if isSTS {
+			r.Reporter.Errorf("Local credentials are not supported for STS clusters")
```

Comment:
Does it mean if the cluster is a rosa sts cluster, the osdCcsAdmin credential is still upserted?

Reply:
STS clusters do not use the `osdCcsAdmin` credentials. Only running `rosa init` creates those creds, but that command is not necessary for STS clusters.